firstvoices/backend/search/utils/constants.py

```diff
@@ -10,6 +10,12 @@
     "operation: %s. %s id: %s"
 )
 
+# Page size
+ES_PAGE_SIZE = 25
+
+# Number of results to return
+ES_MAX_RESULTS = 100
```

Comment:
To clarify, 100 is the maximum page size from the old server, not a cap on the total search results we can return.